js/templates/pageNav.html

```diff
@@ -1,12 +1,51 @@
-<div class="windowControls windowControlsMac">
-  <a class="navClose js-navClose">
-    <i class="ion-ios-close-empty"></i>
-  </a>
-  <a class="navMin js-navMin">
-    <i class="ion-ios-minus-empty"></i>
-  </a>
-  <a class="navMax js-navMax">
-    <i class="ion-ios-plus-empty"></i>
-  </a>
-</div>
-<input type="text" placeholder="Type a handle or OpenBazaar ID" />
\ No newline at end of file
+<header class="borderStacked colBorder">
+  <nav class="bar colPrimary">
+    <div class="flexRow">
+      <div class="windowControlsHolder">
+        <div class="windowControls windowControlsMac">
+          <a class="navClose js-navClose">
+            <i class="ion-ios-close-empty"></i>
+          </a>
+          <a class="navMin js-navMin"></a>
+          <a class="navMax js-navMax"></a>
+        </div>
+      </div>
+      <div>
+        <div class="flexVCent">
+          <a class="iconBtn ion-chevron-left"></a>
+          <a class="iconBtn ion-chevron-right"></a>
+          <a class="iconBtn ion-refresh"></a>
+        </div>
+      </div>
+      <div class="rowDivV colBorderBackground"></div>
+      <div>
+        <div class="flexVCent">
+          <span class="iconBtn">
+            <i class="ion-search"></i>
+          </span>
+        </div>
+      </div>
+      <div class="flexGrow">
+        <div class="flexVCent">
+          <input type="text" class="flexGrow" placeholder="Type a handle or OpenBazaar ID" />
+        </div>
+      </div>
+      <div class="rowDivV colBorderBackground"></div>
+      <div>
+        <div class="flexVCent">
+          <a class="iconBtn ion-android-notifications"></a>
+          <a class="discSm border colBorder"
+          <% if(ob.navimage){ %>
+          //this will be updated to the real image parameter
+          style="background-image: url(ob.navImage), url('imgs/defaultUser.png')"
+          <% } else { %>
+            style="background-image: url('imgs/defaultUser.png')"
```

Comment:
I wasn't sure how we wanted to handle that, I put a copy of the imgs in .tmp, since it seemed kind of weird to have everything else copied there but not the images. 

We can make the path work from .tmp, but we'll have to ignore IDE errors when the editor can't find the path from the template and styles directory (this will also affect images in css).